### ISSUE TITLE
weighted sample contour

### ISSUE THREAD

**Issue by Jane550** [dfm/corner.py#264]
Hi! I have a naive question. Just want to make sure here. If the samples have different weights, the contours in corner.hist2d and corner.corner are drawn based on the weighted samples, right?  Instead of just the samples.

**Comment by dfm** [comment]
Yes, I believe this should work as expected! Please report here if you're finding issues with that.
